phpcs.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="phpList Coding Standard">
+    <description>
+        This standard requires PHP_CodeSniffer >= 3.6.0.
+    </description>
+
+    <arg name="colors"/>
+    <arg name="extensions" value="php"/>
+
+    <!-- The complete PSR-12 ruleset -->
+    <rule ref="PSR12">
```

Comment:
To enforce phpdoc, what about also including `Squiz.Commenting`? See also how it is [configured for WordPress-Docs](https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress-Docs/ruleset.xml).

Reply:
I, too, have lots of ideas which sniffs to add. :-)

Please let's start with PSR-12, get the included sniffs green, and then add other sniffs one by one (with the necessary code changes). Deal?

Reply:
Deal :)